docs/eventing/brokers/broker-types/kafka-broker/README.md

```diff
@@ -386,6 +387,56 @@ The supported consumer delivery guarantees are:
 
 `unordered` is the default ordering guarantee.
 
-### Additional information
+## Data plane Isolation vs Shared Data plane
+
+When using the Broker class `Kafka`, the Knative Kafka Broker uses a shared data plane, which consists of shared `kafka-broker-receiver` and `kafka-broker-dispatcher` deployments for all Kafka Brokers in the cluster.
+
+However, when `KafkaNamespaced` is set as the Broker class, Kafka broker controller creates a new data plane for each namespace that there is a broker exists.
```

Comment:
should we explicitly say, that the DP on a namespace spans across multiple brokers in that NS?